Dapper.GraphQL/TableHelper.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Dapper.GraphQL
+{
+    /// <summary>
+    /// Helper for table related operations.
+    /// </summary>
+    public static class TableHelper
```

Comment:
Totally up to you. I just followed the pattern you'd used for the PropertyHelper, but I agree it'd be nice to have the flexibility to let others supply their own implementation if need be. 

Should we leave the caching up to their implementation or have a base class that can/must be used that does it?

Reply:
I started playing around with this and the problem is all the static methods that create Insert/Query/Delete/Update contexts would need to accept an INameService because it can't be automatically resolved. Potentially this needs a bigger shift to use DI completely to new up these contexts. Have stopped for now as I'd like to hear your thoughts.

Reply:
I'll see if I can take a look over the weekend.  Thanks Ben!

Reply:
Was just poking around Dapper.Contrib and I wonder if it's something we can depend on our take advantage of. The do some stuff around table name mapping. 

https://github.com/StackExchange/Dapper/blob/f29981a77b27a8b4ae1fdb674a081da2fa2ef19f/Dapper.Contrib/SqlMapperExtensions.cs#L39

https://github.com/StackExchange/Dapper/blob/f29981a77b27a8b4ae1fdb674a081da2fa2ef19f/Dapper.Contrib/SqlMapperExtensions.cs#L283

Reply:
Looks like they have a default implementation and then a delegate you can supply to override the default if needed. We could look at an approach like that instead of dependency injection potentially.

Reply:
Hello,

Why is it still not merged? 
I also want to have an ability to specify table name because the default table name is not always suitable.

Reply:
Need to change code to use Dapper.Contrib functionality?